lib/features/home/view_model/get_properties_state.dart

```diff
@@ -0,0 +1,44 @@
+part of 'get_properties_cubit.dart';
+
+abstract class GetPropertiesState extends Equatable {
+  const GetPropertiesState();
+
+  bool get isInitialState => this is GetPropertiesInitialState;
+  bool get isGettingPropertiesState => this is GettingPropertiesState;
+  bool get isGetPropertiesSucceed => this is GetPropertiesSucceedState;
+  bool get isGetPropertiesFailed => this is GetPropertiesFailedState;
+
+  List<PropertyItemData> get properties =>
+      (this as GetPropertiesSucceedState).propertyList;
+
+  String get errorFetching => (this as GetPropertiesFailedState).error;
+
+  @override
+  List<Object?> get props => [];
+}
+
+class GetPropertiesInitialState extends GetPropertiesState {
+  const GetPropertiesInitialState();
+}
+
+class GettingPropertiesState extends GetPropertiesState {
+  const GettingPropertiesState();
+}
+
+class GetPropertiesSucceedState extends GetPropertiesState {
+  final List<PropertyItemData> propertyList;
+
+  const GetPropertiesSucceedState(this.propertyList);
+
+  @override
+  List<Object?> get props => [properties];
```

Comment:
prop => properties;

Reply:
@suesitran Can you explain more about it ? Because props return List<Object>, even though your suggestion does not cause syntax error because properties is a List. But, it would cause some confusion and break the consistent

Reply:
`List<Object?> get props => [properties];` is a list with 1 single object, which is a `List<PropertyItemData>`. When bloc checking prop value, it compares a single instance of this list with another list, and will not notify changes if items in this list are updated.

In the other hand, `List<Object?> get props => properties` is a list in which all properties are checked during bloc prop check, so if the list size change, or any `PropertyItemData` in the list has updated value, bloc will notify the change immediate. In order for this to work properly, `PropertyItemData` also needs to extend `Equatable`